static/builder/unitBuild.js

```diff
@@ -262,6 +265,8 @@ class UnitBuild {
                 }
             } else if (formula.value.mecanism == "hybrid") {
                 this.addToInvolvedStats(["weaponElement","physicalKiller","meanDamageVariance", "atk", "mag"]);
+            } else if(formula.value.mecanism == "summonerSkill"){
```

Comment:
was it needed ? There is already a map for summonerSkill involvedstats on the top of this class